vllm/config.py

```diff
@@ -246,7 +246,9 @@ def _verify_quantization(self) -> None:
             "awq_marlin", "fbgemm_fp8", "compressed_tensors",
             "compressed-tensors", "experts_int8"
         ]
-        tpu_supported_quantization = ["tpu_int8"]
+        tpu_supported_quantization = [
+            "tpu_int8", "compressed_tensors", "compressed-tensors"
```

Comment:
`"compressed_tensors"` is repeated, is that intentional?

Reply:
its repeated in all the others 🫤

Reply:
oh I see `-` vs `_` my bad, looks good 👍